projects/ziti-console-lib/src/lib/features/card-list/card-list.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="fullModal opened">
+    <div class="title" style="filter: brightness(50%)">Create A New Service</div>
+    <div class="subtitle">Choose a Service type</div>
+    <div class="card-list-container">
+        <lib-card
+            (click)="cardSelected('simple')"
+            [imageUrl]="'/assets/svgs/Simple.svg'"
+            [title]="'Simple Service'"
+            [content]="'Create a basic SDK, Endpoint, or Edge Router Hosted Service. Works with older versions of Endpoint Software.'"
```

Comment:
This whole thing needs new words. Maybe something like: 

> Simple services have fewer fields to fill out, only asking for the most commonly used fields.

Reply:
Updated